doc/nrf/ug_nrf5340.rst

```diff
@@ -261,7 +261,7 @@ If you want to enable the network core anyway, set the :kconfig:`CONFIG_BOARD_EN
 
 .. _ug_nrf5340_multi_image:
 
-Single image vs. multi-image build
+Single image and multi-image build
```

Comment:
This is changing the meaning. The two types of build are mutually exclusive, hence "vs." (versus). Plain "and" does not convey this clearly.

Reply:
Since the content mentions more about the Multi-image build, we could change the title to 'Multi-image build'.

Reply:
It mentions only the multi-image build. Not a word about single-image.

Reply:
> we could change the title to 'Multi-image build'.

sounds reasonable!